### PR TITLE
Refactor/delegate emailer to authservice

### PR DESCRIPTION
**Pull request by johnnyaug**

### CODE REVIEW

File: api/authorization.yml [hunk 1]
```diff
@@ -156,6 +156,8 @@ components:
           format: byte
         external_id:
           type: string
+        invite:
```
Comment (arielshaqed) [inline api/authorization.yml:159]: Please document this field. It is not clear to me whether this is imperative or descriptive.

File: pkg/api/login_handler.go [hunk 2]
```diff
@@ -19,17 +19,15 @@ type LoginResponseData struct {
...
+	DefaultResetPasswordExpiration = 20 * time.Minute
 	// Deprecated: use gorilla managed session
 	JWTCookieName           = "access_token"
 	InternalAuthSessionName = "internal_auth_session"
 	TokenSessionKeyName     = "token"
 	OIDCAuthSessionName     = "oidc_auth_session"
 
-	LoginAudience         = "login"
-	ResetPasswordAudience = "reset_password"
+	LoginAudience = "login"
```
Comment (arielshaqed) [inline pkg/api/login_handler.go:30]: The fact that the reset audience definition had to move is probably a sign that the login audience should also move over. (Not this PR, just an observation)

**Review by arielshaqed — APPROVED** [review]
Neat! Looks like a much better place for this logic.
Thanks...

**Comment by arielshaqed** [comment]
I am curious: I was under the impression that we would be removing email-based login in favour of OIDC.  If so, then why are we improving this code when we could be working towards removing it?

File: api/authorization.yml (hunk 1 above)
Comment (johnnyaug) [reply api/authorization.yml:159]: Done

**Comment by johnnyaug** [comment]
Regarding @arielshaqed's comment, we synced f2f, writing here to document: While we may want to phase out email-based login, this wasn't yet decided upon.

**Pull request merged**